pages/lists/[listId]/activity.tsx

```diff
@@ -166,7 +178,9 @@ const ListActivityPage = ({ list, numberOfContributors, isError, activityData, i
             data={treemapData}
             color={getGraphColorPalette()}
           />
-          <ContributionsEvolutionByType data={evolutionData} isLoading={isLoadingEvolution} />
+          <FeatureFlagged flag="contributions_evolution_by_type" featureFlags={featureFlags}>
```

Comment:
This will only render if the `contributions_evolution_by_type` feature flag is enabled for the currently logged in user.

Reply:
The data is still available, but I didn't go that granular for the feature flag as we're working on the graph at the moment and there's no risk in the data being present in the page.